### DISCUSSION TITLE
无法运行的情况

### DISCUSSION THREAD

**Discussion by Tulip129** [LmeSzinc/AzurLaneAutoScript#1330] [General]
点击运行后提示我
INFO     23:09:38.770 │ <<< MINITOUCH INIT >>>                               
INFO     23:09:38.790 │ [Device Orientation] 0 (Normal)                      
ERROR    23:09:38.794 │ AdbError: unknown host service                       
CRITICAL 23:09:38.796 │ Retry click_minitouch() failed                       
CRITICAL 23:09:38.799 │ Request human takeover   
这是咋回事啊

**Comment by LmeSzinc** [comment]
雷电模拟器专属问题，不使用雷电模拟器就解决了

**Comment by lzy0702** [comment]
minitouch会莫名其妙挂掉，把模拟点击换成adb就行
